components/brave_rewards/resources/page/reducers/rewards_reducer.ts

```diff
@@ -19,17 +19,14 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       break
     }
     case types.TOGGLE_ENABLE_MAIN: {
-      if (state.initializing) {
+      if (state.initializing && state.enabledMain) {
         break
       }
 
       state = { ...state }
       const key = 'enabledMain'
       const enable = action.payload.enable
-
-      if (enable) {
```

Comment:
I don't see us removing this `if` guard in the android page, unless I'm just missing it...

Reply:
removed on android page as well